README.md

```diff
@@ -6,6 +6,46 @@ Production instance: http://logdetective.com
 
 ![Video](./frontend/public/img/log_detective_demo.gif)
 
+## Submit annotations via API
+
+### Copr
+
+This is how you can submit annotations to Log Detective via our API. We'll use python code to demonstrate this.
+```
+import requests
+
+build_id = "8896521"
+chroot = "rhel-9-x86_64"
+url = f"https://logdetective.com/frontend/contribute/copr/{build_id}/{chroot}"
+# url = f"http://localhost:5020/frontend/contribute/copr/{build_id}/{chroot}"
+data = {
+    "username": "FAS:me",
+    "fail_reason": "Failed because...",
+    "how_to_fix": "Like this...",
+    "spec_file": {
+        "name": "llvm.spec",
+        "content": "Yes, actual content of the spec file"
+    },
+    # if you are annotating a container build
+    # "container_file": None,
+    "logs": [
+        {
+            "name": "build.log",
+            "content": "content of the build log",
+            "snippets": [
+                {
+                    "start_index": 1,
+                    "end_index": 2,
```

Comment:
I assume these are line indexes. An index typically starts at 0 but if it is a line number I assume it starts at 1. Please clarify.

Reply:
I recall they are character indexes.
@FrostyX @nikromen can you help me answer here please? I'll update the docs then.

Reply:
Yes, exactly. This:

```
data["logs"][0]["content"][start_index:end_index+1]
```

should be equal to the `text` field.